.github/workflows/macos.yml

```diff
@@ -0,0 +1,28 @@
+name: macos
```

Comment:
Seems like github charges for mac runners.
So, the question is if this should be included and disabled for now or simply leave it out.

Reply:
I am not really willing to pay for it, as it is not that important enough.

Reply:
Removed